src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -1,15 +1,21 @@
 import {
 	Component,
 	HostBinding,
-	Input
+	Input,
+	Output,
+	EventEmitter,
+	Optional
 } from "@angular/core";
+import { DomSanitizer } from "@angular/platform-browser";
+import { Router } from "@angular/router";
 
 @Component({
 	selector: "ibm-breadcrumb-item",
 	template: `
 	<a
 		class="bx--link"
 		href="{{skeleton ? '/#' : href}}"
```

Comment:
There's some warns around this
```suggestion
		[href]="(skeleton ? '/#' : href)"
```